electroncash/commands.py

```diff
@@ -33,7 +33,7 @@
 import time
 
 from decimal import Decimal as PyDecimal  # Qt 5.12 also exports Decimal
-from functools import wraps
+from functools import wraps, partial
```

Comment:
```suggestion
from functools import wraps
```

Rationale: Make the diff smaller :)